ctam/interfaces/fw_update_ifc.py

```diff
@@ -268,6 +247,8 @@ def ctam_stage_fw(
                 StageFWOOB_Status = False
         if return_task_id:
             return StageFWOOB_Status, FwUpdTaskID
+        elif return_msg:
+            return StageFWOOB_Status, stage_msg, FwUpdTaskID
```

Comment:
What if both return_task_id and return_msg are true? We should handle all possible scenarios.

Reply:
This issue has been resolved.